### PR TITLE
(chore) Cache file size workflow deps

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

This PR introduces dependency caching for the file size impact workflow.

### CODE REVIEW

File: .github/workflows/size.yml [hunk 1]
```diff
@@ -11,11 +11,23 @@ jobs:
...
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+      
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: npx lerna bootstrap
```
Comment (brandones) [inline .github/workflows/size.yml:29]: This should be replaced with `yarn`, I think

**Review by brandones — APPROVED** [review]

File: .github/workflows/size.yml (hunk 1 above)
Comment (ZacButko) [reply .github/workflows/size.yml:29]: @denniskigen  is this possible?

**Comment by vasharma05** [comment]
Any updates on this @denniskigen?

**Comment by makombe** [comment]
@denniskigen Do we still need this PR? if so please resolve the conflicts. Thanks

**Comment by denniskigen** [comment]
Closing as I'm not entirely sure about the utility of this change at this present moment.

**Pull request closed without merging**